src/Eyes/Url/UrlStatus.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Url;
+
+use BeholderWebClient\Eyes\Status;
+
+class UrlStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: php lib curl is not installed.';
+
+  const COULD_NOT_CONNECT = 'Could not connect to url: ';
+  const COULD_NOT_CONNECT_NUMBER = 600;
+
+  const STATUS_CODE_WAS_NOT_EXPECTED = 'The returned status code was not the expected, received: ';
```

Comment:
I didn't understand a thing of what you are saying.
The returned status code was not the expected
vs
The returned status code was not the expected

The returned status code was not expected, didn't understand why you have to put "the" since you explicitly point that what was not expected was the returned http status code.

"and what happen was not it.", did you mean "and that wasn't what was happening" ?